f8a_worker/workers/init_analysis_flow.py

```diff
@@ -37,6 +39,15 @@ def execute(self, arguments):
         # make sure we store package name in its normalized form
         arguments['name'] = normalize_package_name(ecosystem.backend.name, arguments['name'])
 
+        # Dont try ingestion for private packages
```

Comment:
Which ecosystems were problematic? We have some code that should deal with non-existent versions in https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/f8a_worker/process.py#L349-L361 (this one's for NPM).

So maybe there is a bug somewhere?

Reply:
this is just to prevent any pvt package from ingestion. we dont even go to the extent of checking the version

Reply:
Also, having this later in the flow, the queue gets cluttered